docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om eigenschapinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype de eigenschap is opgenomen. |
+| - gevolg       | Een overzicht van eigenschapinformatie van het zaaktype wordt teruggegeven.            |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om details van eigenschapinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype de eigenschap is opgenomen. Tevens moet de identficatie van de eigenschap welke uitgevraagd wordt bekend zijn. |
+| - gevolg       | Detailinformatie van de eigenschap wordt teruggegeven.            |
+| Gegevens       | Objecttype EIGENSCHAP inclusief de relatie naar ZAAKTYPE<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/eigenschap_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+      
+### Resource: Roltypen
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil alle roltypen van een zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.            |
+| *Toevoegen*    | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  | Nog niet beschikbaar.     |
+| - voorwaarde   | Nog te bepalen.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om roltypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het roltype is opgenomen. |
+| - gevolg       | Een lijst met roltypeinformatie behorend bij het zaaktype wordt teruggegeven aan de consumer.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om roltypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype het roltype is opgenomen. Tevens moet de identficatie van het roltype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Roltypeinformatie wordt teruggegeven aan de consumer.             |
+| Gegevens       | Objecttype ROLTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
```

Comment:
Geen relatie naar het Zaaktype waar het bijhoort?

Reply:
Correct, is toegevoegd.